stages/diagram-recognition/src/main/kotlin/edu/kit/kastel/mcse/ardoco/lissa/diagramrecognition/informants/OcrInformant.kt

```diff
@@ -15,87 +18,150 @@ import org.apache.hc.core5.net.URIBuilder
 import java.io.ByteArrayInputStream
 import java.io.InputStream
 import java.net.URI
-import java.util.*
-
-class OcrInformant(dataRepository: DataRepository) : ImageProcessingDockerInformant(
-    DOCKER_OCR,
-    DEFAULT_PORT,
-    DOCKER_OCR_VIA_DOCKER,
-    ID,
-    dataRepository,
-    "ocr"
-) {
+import java.util.SortedMap
+
+class OcrInformant(
+    dataRepository: DataRepository
+) : ImageProcessingDockerInformant(
+        DOCKER_OCR,
+        DEFAULT_PORT,
+        DOCKER_OCR_VIA_DOCKER,
+        ID,
+        dataRepository,
+        "ocr"
+    ) {
     companion object {
         const val ID = "OCRInformant"
         const val DOCKER_OCR = "ghcr.io/lissa-approach/diagram-ocr:latest"
         const val DEFAULT_PORT = 5005
-        const val DOCKER_OCR_VIA_DOCKER = true
+        const val DOCKER_OCR_VIA_DOCKER = false
```

Comment:
Please revert

Reply:
Id like to also set this with some sort of environment variable. Maybe this can even be inferred from context? This is true in all cases except when REMOTE_DOCKER_URI is set, right? If that's the case, Ill just check that env var to determine the appropriate value